circle.yml

```diff
@@ -8,7 +8,7 @@ deployment:
       - 'go get github.com/tools/godep'
       - 'godep restore'
       - 'godep update ./...'
-      - 'sed "s/\[your-app-name\]/$WHITEBOARDBOT_CF_APP_NAME/" manifest.yml.example > manifest.yml'
+      - 'grep -v "^      " manifest.yml.example | grep -v env: | sed "s/\[your-app-name\]/$WHITEBOARDBOT_CF_APP_NAME/" > manifest.yml'
```

Comment:
Added this to strip the `env:` section from `manifest.yml.example` since all these environment variables are probably set in Cloud Foundry already and we don't necessarily want to have to set all these in Circle CI as well.